### PR TITLE
mocking out critical infrastructure

### PR DESCRIPTION
**Pull request by HenryGeorgist**
wip critical infrastructure

### CODE REVIEW

**Review by HenryGeorgist — COMMENTED** [review]
looks really good. please remove any features that have a root url problem, cant get geojson,  or a 404 problem. Document the issues in comments so we can keep track of things that arent working like expected. Remove the items from the 3 string arrays that correspond with any removed features.

**Comment by marko-nedzbala** [comment]
I also placed this on our MS Teams CHARTS chat, but also placing this here.
I decided to check the features for their geometry types. My main concern is 2 of the previously working links changed their URLs in a matter of days. Just something we should all be aware of.

Layer Emergency Medical Service (EMS) Stations changed to carto.nationalmap.gov
Layer Fire Terminals changed to returning a 404 error.

File: criticalinfrastructure/hsip-api.go [hunk 1]
```diff
@@ -32,53 +32,74 @@ const (
...
+	CellularTowers        //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Cellular_Towers_New/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	DialysisCenters       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Dialysis_Centers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	EPAandFRSPowerPlants  //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Environmental_Protection_Agency_EPA_Facility_Registry_Service_FRS_Power_Plants/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	FacilityInterests     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Facility_Interest/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	GeneratingUnits       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/GeneratingUnits1/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
 	// Hospitals
```
Comment (HenryGeorgist) [inline criticalinfrastructure/hsip-api.go:42]: im not sure why hospitals got commented out - it should work.

File: criticalinfrastructure/hsip-api.go [hunk 2]
```diff
@@ -32,53 +32,74 @@ const (
...
-	MicrowaveServiceTowers                         //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Microwave_Service_Towers_New/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	NationalShelterSystemFacilities                //https://gis.fema.gov/arcgis/rest/services/NSS/FEMA_NSS/FeatureServer/5/query?outFields=*&where=1%3D1&f=geojson
-	NursingHomes                                   //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/NursingHomes/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	PagingTransmissionTowers                       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Paging_Transmission_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	Pharmacies                                     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Paging_Transmission_Towers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	HurricaneEvacuationRoutes              //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Hurricane_Evacuation_Routes/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
```
Comment (HenryGeorgist) [inline criticalinfrastructure/hsip-api.go:43]: are Hurricane Evacuation routes linear features?

File: criticalinfrastructure/hsip-api.go (hunk 1 above)
Comment (marko-nedzbala) [reply criticalinfrastructure/hsip-api.go:42]: It was already there, same with PowerPlants, but put there to keep the order.

File: criticalinfrastructure/hsip-api.go (hunk 2 above)
Comment (marko-nedzbala) [reply criticalinfrastructure/hsip-api.go:43]: According to the JSON they are LineString.

**Pull request merged**